acceptancetests/assess_deploy_webscale.py

```diff
@@ -139,13 +154,23 @@ def main(argv=None):
         raw_logs = extract_module_logs(client, module=args.logging_module)
         timings = extract_txn_timings(raw_logs, module=args.logging_module)
         # Calculate the timings to forward to the datastore
-        total_time = calculate_total_time(timings)
-        total_txns = len(timings)
-        max_time = calculate_max_time(timings)
-        since = (time.time() - begin)
+        metrics = makeMetrics(
+            calculate_total_time(timings),
+            len(timings),
+            calculate_max_time(timings),
+            (time.time() - begin),
+        )
 
-        log.info("The timings for deployment: total txn time: {}, total nums txn: {}, max txn time: {}, max time (seconds): {}".format(total_time, total_txns, max_time, since))
+        log.info("Metrics for deployment: {}".format(metrics))
+
+        rclient = reportingClient(args.reporting_uri)
+        rclient.report(metrics, tags={
+            "git-sha": args.git_sha,
+            "charm-bundle": args.charm_bundle,
+        })
+        log.info("Metrics successfully sent to report storage")
```

Comment:
Another were maybe this should only be output if we know report worked where here it could be run w/o successful report?